test/pages/ViewProgress.scala

```diff
@@ -86,7 +86,7 @@ object ViewProgress extends CommonPage {
 
   def checkMainMessage(mainMessage: String)(implicit webDriver: WebDriver): Assertion = readMainMessage shouldBe mainMessage
 
-  def backExists()(implicit driver: WebDriver): Assertion = idPresent("back-link") shouldBe true
+  def backExists()(implicit driver: WebDriver): Assertion = cssPresent(".govuk-back-link") shouldBe true
```

Comment:
should this be used on the new back link you've put on the view repayment account page to test it's there?

Reply:
what I meant was the view repayment account page wouldn't have had a test for a back link on it previously cos it didn't have a back link, but you've just added one, so shouldn't there be a test for that new back link somewhere? Unless I've just missed something somewhere 😄

Reply:
Oh, I thought there already was one, but now see I misread https://github.com/hmrc/vat-repayment-tracker-frontend/blob/main/test/pages/tests/VrtVatRegistrationCancelledSpec.scala#L30 

I will add something